timesketch/frontend-ng/src/components/LeftPanel/AnalyzerResultTimeline.vue

```diff
@@ -289,12 +289,79 @@ limitations under the License.
           </tbody>
         </v-simple-table>
       </div>
-      <div v-else v-show="expanded" class="ml-3 pb-1 mr-2">
-        <v-icon>mdi-alert-octagon-outline</v-icon>
-        <span class="ml-1">
-          Showing multi analyzer results is not supported in the new UI yet. Please visit the old UI to see these
-          results.
-        </span>
+      <div v-else v-show="expanded"
+        :class="
+          $vuetify.theme.dark
```

Comment:
That logic could go into a method or computed property, no? It would declutter the template a bit.